### PR TITLE
[MOBILE-1048] update iOS to 13.0.4

### PR DESCRIPTION
**Pull request by crow**
### What do these changes do?
Updates flutter iOS version to 13.0.4

### How did you verify these changes?
Manually tested sample

#### Verification Screenshots:
flutter-ios-13 0 4

### CODE REVIEW

File: lib/airship.dart [hunk 1]
```diff
@@ -1,4 +1,4 @@
-library airship;
+library airship_flutter;
```
Comment (rlepinski) [inline lib/airship.dart]: is this the standard naming convention?

Comment (crow) [reply lib/airship.dart]: yes

File: example/ios/Podfile [hunk 2]
```diff
@@ -62,6 +62,9 @@ target 'Runner' do
   }
 end
 
+# Prevent Cocoapods from embedding a second Flutter framework and causing an error with the new Xcode build system.
```
Comment (BrianBatchelder) [inline example/ios/Podfile]: Is this a cocoa pods bug?

File: ios/Classes/AirshipInboxMessageView.swift [hunk 3]
```diff
@@ -60,27 +61,36 @@ class AirshipInboxMessageView : NSObject, FlutterPlatformView {
...
+                             details:nil))
+       }
     }
 
     func view() -> UIView {
         return webView
     }
+
+    func close() {
+        // In the future we should send a close event to the dart layer
```
Comment (BrianBatchelder) [inline ios/Classes/AirshipInboxMessageView.swift]:
```suggestion
        // REVISIT: we should send a close event to the dart layer
```

File: example/ios/Podfile (hunk 2 above)
Comment (crow) [reply example/ios/Podfile]: its actually a bug in xcode's new build system afaik

**Pull request merged**